install_config/storage_examples/external_gluster_dynamic_example.adoc

```diff
@@ -0,0 +1,415 @@
+
+GlusterFS Cluster running RHGS 3.1 (3 nodes, each with at least 2 X 100GB RAW devices)
+
+	   - gluster23.rhs (192.168.1.200)
+	   - gluster24.rhs (192.168.1.201)
+	   - gluster25.rhs (192.168.1.202)
+
+Heketi Server/Client Node running RHEL 7.x or RHGS 3.1 (Heketi can be installed on one of the Gluster Nodes)
+
+	   - glusterclient2.rhs (192.168.1.203)
+
+{product-title} Node
+
+	   - k8dev2.rhs (192.168.1.208)
+
+[NOTE]
+====
+This example uses a single node cluster (master + node)
+====
+
+[[installing-and-configuring-Heketi]]
+== Installing and configuring Heketi
+
+Heketi is used to manage our gluster cluster storage (adding volumes, removing volumes, etc…). As stated this can be RHEL or RHGS,
+and can be installed on one of the existing Gluster Storage Nodes.
+This example will use a stand-alone RHGS 3.1 Node running Heketi.
+
+The link:https://access.redhat.com/documentation/en-US/Red_Hat_Storage/3.1/html/Administration_Guide/ch06s02.html[Red Hat Storage Administration Guide] can be used a reference
+during this process.
+
+
+. Install Heketi and the Heketi-Client.
++
+----
+	From the node designated to run Heketi and Client.
+
+	# yum install heketi heketi-client -y
+----
++
+[NOTE]
+====
+The Heketi Server can be any of the existing nodes, typically this will be the Master Node.
+For this example however, we used a separate Node not part of the GlusterFS or {product-title} cluster.
+====
+
+. Create and install Heketi private keys on each cluster node.
++
+----
+	From the Node that is running Heketi.
+
+ 	# ssh-keygen -f /etc/heketi/heketi_key -t rsa -N ''
+ 	# ssh-copy-id -i /etc/heketi/heketi_key.pub root@gluster23.rhs
+	# ssh-copy-id -i /etc/heketi/heketi_key.pub root@gluster24.rhs
+	# ssh-copy-id -i /etc/heketi/heketi_key.pub root@gluster25.rhs
+ 	# chown heketi:heketi /etc/heketi/heketi_key*
+----
+
+
+. Edit the /usr/share/heketi/heketi.json file to setup the SSH executor.
++
+----
+	Below is an excerpt from the /usr/share/heketi/heketi.json file, the part to configure
+	is the executor and ssh section. 
+
+       ...
+	"executor": "ssh",  <1>
+
+	"_sshexec_comment": "SSH username and private key file information",
+	"sshexec": {
+  	  "keyfile": "/etc/heketi/heketi_key",  <2>
+  	  "user": "root",  <3>
+  	  "port": "22",  <4>
+  	  "fstab": "/etc/fstab"  <5>
+	},
+----
+<1> Change "executor": from mock to ssh
+<2> Add in the public key directory specified in previous step
+<3> Update SSH user (should have sudo/root type access)
+<4> Set the port to 22 - remove all other text
+<5> Set the fstab to default (etc/fstab) - remove all other text
+
+
+. Restart and enable.
++
+----
+	# systemctl restart heketi
+	# systemctl enable heketi
+----
+
+. Test connection to Heketi.
++
+----
+        # curl http://glusterclient2.rhs:8080/hello
+        Hello from Heketi
+----
+
+. Set an environment variable for the Heketi Server.
++
+----
+	# export HEKETI_CLI_SERVER=http://glusterclient2.rhs:8080/hello
```

Comment:
Should be without "hello": export HEKETI_CLI_SERVER=http://glusterclient2.rhs:8080/hello